Osteppy/src/EODreminder.js

```diff
@@ -54,52 +57,50 @@ module.exports.sendDM = (RA, message) => {
 const resetRAList = () => {
   execSync(cpCommand);
 };
-// resetRAList();
+
 
 // Reminder schedule
 // Josue: 4PM, 9PM
 // Miguel: 9PM, 8:30AM, 4:30PM
-// Daniel: 9PM
 // Ian: 9PM, 4:30PM
 // Pouya: 10PM
-// Yan: 9:30PM, 4:55PM
 // Olga: 8:30PM
+// Stuart: 9PM
+// Raymond: 9AM
 
 // Checks the time to send custom EOD reminders
 const checkTime = (clock) => {
   if (clock.second === 0) {
     if (clock.minute === 0) {
```

Comment:
This big if-else chain needs refactoring. RA names and messages should also not be hard coded. Consider creating an array of RA objects that have names, reminder times, and messages. Then iterate through that.